core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -183,11 +185,13 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
```

Comment:
I think in this test we don’t want to reinitialize `sTree` for each test case. That being said, using `beforeClass` makes it a bit more efficient.